source/includes/compute/_templates.md

```diff
@@ -16,39 +16,59 @@ curl -X GET \
     "recordCount": 1
   }
 }
 ```
 
-<code>GET /services/<a href="#service-connections">:service_code</a>/<a href="#environments">:environment_name</a>/templates</code>
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/templates</code>
 
-Retrieve a list of all templates of an [environment](#environments) It will include all the public templates of the system.
+Retrieve a list of all templates of an [environment](#administration-environments) It will include all the public templates of the system.
 
 Attributes | &nbsp;
 ---------- | -----
 `id`<br/>*UUID* | The id of the template
 `name`<br/>*string* | The name of the template
 `description`<br/>*string* | The description of the template
+`defaultUsername`<br/>*string* | The default username of the template
 `size`<br/>*long* | The size of the template in bytes
-`isPublic`<br/>*boolean* | true if public to everyone. Your custom templates will always be private
-`isReady`<br/>*boolean* | true if the template is ready to be used for a new [instance](#instances)
-`isDynamicallyScalable`<br/>*boolean* | true if you can dynamically scale an [instance](#instances) with this template
+`availablePublicly`<br/>*boolean* | **true** if public to everyone. Your custom templates will always be private
+`ready`<br/>*boolean* | **true** if the template is ready to be used for a new [instance](#cloudstack-instances)
+`dynamicallyScalable`<br/>*boolean* | **true** if you can dynamically scale an [instance](#cloudstack-instances) with this template
+`extractable`<br/>*boolean* | **true** if you want the template to be downloadable from a generated URL
 `created`<br/>*string* | The creation date of the template
-`osType`<br/>*string* | The OS type of the template (e.g. Ubuntu, CentOS...)
-`availableInZones`<br/>*array[UUID]* | List of all [zone ids](#zones) that the template is available in
+`osTypeId`<br/>*UUID* | Id of the OS type
+`osTypeName`<br/>*string* | The OS type of the template (e.g. Ubuntu, CentOS...)
+`zoneId`<br/>*UUID* |The [zone id](#cloudstack-zones) that the template is available in. Empty if available in multiple zones
+`zoneName`<br/>*string* |The [zone id](#cloudstack-zones) that the template is available in. Empty if available in multiple zones
+`availableInZones`<br/>*array[UUID]* | List of all [zone ids](#cloudstack-zones) that the template is available in
+`hypervisor`<br>*string* | The name of the hypervisor
+`format`<br>*string* | The template format for the chosen hypervisor
+`nicAdapter`<br>*string* | `VMWare only` The network interface controller name. Empty if not selected.<br>*Values*: e1000, pcnet32, vmxnet2 or vmxnet3.
+`rootDiskController`<br>*string* | `VMWare only` The root disk controller name. Empty if not selected.<br>*Values*: ide, scsi for CloudStack version up to 4.4.<br>*Values*: ide, scsi, osdefault, pvscsi, lsilogic, lsisas1068, buslogic for CloudStack version 4.5 and up.
```

Comment:
VMWare isn't supported in cca so I would remove all fields relative to it

Reply:
Removed.